bridge/slack/handlers.go

```diff
@@ -187,6 +192,7 @@ func (b *Bslack) handleStatusEvent(ev *slack.MessageEvent, rmsg *config.Message)
 		rmsg.Username = sSystemUser
 		rmsg.Event = config.EVENT_JOIN_LEAVE
 	case sChannelTopic, sChannelPurpose:
+		b.populateChannels()
```

Comment:
This must be up-to-date since we check for "[nosync]" suffix later

Reply:
I would rather add the checking for ` [nosync]` here.

Reply:
Ok, so we're not going to be doing `nosync` anymore, so technically I don't need this, but doesn't it make sense to keep the state of `ChannelInfo` up-to-date with purpose/topic? So I wonder if we should just run this on changes that knock it out of sync (like these), nevermind whether we're _yet_ deciding to process it.

If we're not updating, the inconsistency will likely hit someone else later who assumed ChannelInfo to be up-to-date.

tl;dr Let's leave this in to keep our internal representations matching reality

Reply:
Yes. Do keep it.